addon/webextension/selector/ui.js

```diff
@@ -286,7 +286,9 @@ window.ui = (function () { // eslint-disable-line no-unused-vars
     hide: function () {
       window.removeEventListener("scroll", this.onScroll, false);
       window.removeEventListener("resize", this.onResize, true);
-      this.element.style.display = "none";
+      if (this.element) {
```

Comment:
I'd guess this could happen if the overlay is toggled on and off quickly, so that it's torn down before the elements were rendered into the DOM